### DISCUSSION TITLE
How are you handling the Value Based Care Specialty Capitation in the MRF?

### DISCUSSION THREAD

**Discussion by tdfow** [CMSgov/price-transparency-guide#741] [Q&A]
From the mandate, it seem there is a reference around value-based purchasing models, where the plan/issuer must provide the 'base' negotiated rate for the in-network file but these rates may be adjusted based on contractual goals ---- so is the additional information field just being used to state the disclaimers for the rate being adjusted based on quality, volume and efficiency of care?

If you have value-based care specialty capitation arrangements, how are you handling the posting of the rates in the MRF?

**Comment by shaselton-usds** [comment]
Hi @tdfow 

That would be ideal to leverage the additional information attribute for the type of information/data that you're referencing. If you find that there is a lot of repeated information in that attribute as you build out these MRFs, that would be great feedback to think through how that data could be structured and incorporated into the schema.

Thanks again!
